index.js

```diff
@@ -49,6 +50,13 @@ export async function setupRuby(options) {
 
   setupPath(newPathEntries)
 
+  // When setup-ruby is used by other actions, this allows code in them to run
+  // before 'bundle install'.  Installed dependencies may require additional
+  // libraries & headers, build tools, etc.
+  if (inputs['func'] instanceof Function) {
+    await inputs['func']()
```

Comment:
Should we pass it `{ rubyPrefix, platform, engine, version }`?
Then the callback might use it or not, but it's probably useful at least in some cases.

Reply:
Changed the order to match a code line a few statements later.  Done